src/StartPage/DifficultySelection.js

```diff
@@ -0,0 +1,122 @@
+import React, { Fragment, Component } from 'react';
+import styled from 'styled-components';
+import ClipLoader from "react-spinners/ClipLoader";
+
+const StyledButton = styled.div`
+    border: 2px solid #0094da;
+    border-radius: 0;
+    background-color: #0094da;
+    color: #fff;
+    display: inline-block;
+    font-size: 16px;
+    font-weight: 600;
+    padding: 14px 75px;
+    margin: 0 auto;
+    text-transform: uppercase;
+    transition: color .2s ease, background-color .2s ease;
+    cursor: pointer;
+    margin-top: 20px;
+
+    &:hover {
+        background-color: #fff;
+        color: #0094da;
+      }
+`;
+
+const StyledDiv = styled.div`
+    font-size: 22px;
+    padding-left: 50px 50px 50px 50px;
+    box-sizing: border-box;
+    margin-top: 20px;
+`;
+
+class DifficultySelection extends Component {
+    constructor(props){
+        super();
+        this.state = {
+            selectedOption: 'easy',
+            data: null,
+            loading: null
+        }
+    }
+
+    dataHandler = () => {
+        this.setState({ loading: true });
+        const selectedOption = this.state.selectedOption;
+        console.log(selectedOption);
```

Comment:
I guess this remained here from a debugging session, but not much use of it any more